libs/dal/src/repositories/change/change.entity.ts

```diff
@@ -0,0 +1,23 @@
+export class ChangeEntity {
+  _id?: string;
+
+  _creatorId: string;
+
+  _environmentId: string;
+
+  _organizationId: string;
+
+  _entityId: string;
+
+  enabled: boolean;
+
+  type: ChangeEntityTypeEnum;
+
+  change: any;
+
+  createdAt: string;
+}
+
+export enum ChangeEntityTypeEnum {
+  NOTIFICATIONTEMPLATE = 'NotificationTemplate',
```

Comment:
```suggestion
  NOTIFICATION_TEMPLATE = 'NotificationTemplate',
```
Might be easier to read :)